src/ui/common/src/components/integrations/operatorsOnIntegration.tsx

```diff
@@ -0,0 +1,88 @@
+import Alert from '@mui/material/Alert';
+import AlertTitle from '@mui/material/AlertTitle';
+import Box from '@mui/material/Box';
+import CircularProgress from '@mui/material/CircularProgress';
+import React, { useState } from 'react';
+import { useSelector } from 'react-redux';
+
+import { OperatorsForIntegrationItem } from '../../reducers/integrationOperators';
+import { RootState } from '../../stores/store';
+import { isFailed, isLoading } from '../../utils/shared';
+import { ListWorkflowSummary } from '../../utils/workflows';
+import WorkflowAccordion from '../workflows/accordion';
+
+const OperatorsOnIntegration: React.FC = () => {
+  const listWorkflowState = useSelector(
+    (state: RootState) => state.listWorkflowReducer
+  );
+  const operatorsState = useSelector((state: RootState) => {
+    return state.integrationOperatorsReducer;
+  });
+  const [expandedWf, setExpandedWf] = useState('');
```

Comment:
For this useState, can you add a type for expanedWf please?

Will look something like this:
```
const [expandedWf, setExpandedWf] = useState<string>();
```